wallet/chainntfns.go

```diff
@@ -14,9 +14,16 @@ import (
 	"github.com/decred/dcrutil"
 	"github.com/decred/dcrwallet/chain"
 	"github.com/decred/dcrwallet/waddrmgr"
+	"github.com/decred/dcrwallet/wallet/txauthor"
 	"github.com/decred/dcrwallet/wtxmgr"
 )
 
+const (
+	// ticketPurchaseAttemptsPerBlock is the default number of ticket
+	// purchase attempts per block.
+	ticketPurchaseAttemptsPerBlock = 10
```

Comment:
Is there a reason for lowering this from the old value (when it was maxAttempts)?